algebra-cl-gen/src/gpu/multiexp/field2.cl

```diff
@@ -1,5 +1,3 @@
-// Fp2 Extension Field where u^2 + 1 = 0
```

Comment:
It'd be good to mention what we generalized here (I expect extension field using `u^2 - nonresidue = 0`, where `nonresidue` is a quadratic nonresidue).